### PR TITLE
Fix UI bugs

### PR DESCRIPTION
**Pull request by crux25**
This PR closes issues: #56 #52 #45 and #32
Overall it improves the UI, the goal is to make the app testable.

### CODE REVIEW

**Review by devchoplife — CHANGES_REQUESTED** [review]

File: treasury/types.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+type TreasuryInfo struct {
+	Net string
+
+	// Page parameters
+	MaxTxLimit    int64
+	Path          string
+	Limit, Offset int64  // ?n=Limit&start=Offset
+	TxnType       string // ?txntype=TxnType
+
+	// TODO: tadd and tspend can be unconfirmed. tspend for a very long time.
```
Comment (devchoplife) [inline treasury/types.go:59]: Do we need to create an issue for this?

Comment (crux25) [reply treasury/types.go:59]: No we don't to, that comment is from upstream. I left it there for reference purpose.

File: treasury/types.go [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	tStr = strings.Trim(tStr, `"`)
+	T, err := time.Parse(timeDefFmtJS, tStr)
+	if err != nil {
+		return err
+	}
+	t.T = T
+	return nil
+}
+
+// PrettyMDY formats the time down to day only, using 3 day month, unpadded day,
```
[the remark this answers is not in the document]
Comment (crux25) [reply treasury/types.go]: Resolved

File: treasury/types.go [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+func (t TimeDef) RFC3339() string {
+	return t.T.Format(timeDefFmtJS)
+}
+
+// UNIX returns the UNIX epoch time stamp.
+func (t TimeDef) UNIX() int64 {
+	return t.T.Unix()
+}
+
+func (t TimeDef) Format(layout string) string {
```
[the remark this answers is not in the document]
Comment (crux25) [reply treasury/types.go:104]: resolved

**Review by devchoplife — APPROVED** [review]
Looks good

**Pull request merged**